docker/main/ngen/Dockerfile

```diff
@@ -664,6 +664,10 @@ RUN if [ "${NGEN_WITH_PYTHON}" == "ON" ]; then \
     fi
 USER ${USER}
 
+ENV VIRTUAL_ENV=/dmod/venv
+RUN python3 -m venv $VIRTUAL_ENV && pip3 install numpy
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```

Comment:
Hmmm, I assume you are adding this to `PATH` so `venv` is effectively always activated. Is there a reason why we can't use the default python environment _or_ instead add `venv` activation in the entrypoint?

Reply:
Basically, something weird seems to be going on, probably after #755, that's breaking the image build.  See https://github.com/CIROH-UA/NGIAB-HPCInfra/issues/12 and https://github.com/CIROH-UA/NGIAB-CloudInfra/issues/137 for more details on this specifically, as others are running into it also.

Reply:
Thanks for linking those related issues. Seems like something to do with using a non-virtual environment. My main concern is making it really clear that we are using a virtual environment in the image without needing to come and read the `Dockerfile`.

Reply:
Fair point.  I've opened #630.  I don't really want to move away from the default environment, but it seems necessary.  But I'd like to eventually take this out, once the underlying ngen build issues are resolved.

Reply:
Yeah, this is super weird. Lets just move ahead like you are suggesting and revisit this when we have a fix.